components/raftstore/src/store/worker/split_check.rs

```diff
@@ -810,6 +825,8 @@ impl<EK: KvEngine, S: StoreHandle> Runner<EK, S> {
                 }
             }
 
+            split_keys = host.split_keys();
```

Comment:
```suggestion
            let maybe_splittable = !host.split_keys().is_empty();
```

Reply:
host.split_keys() is not reentrant (only called once)